hooks/cart/useCart.js

```diff
@@ -37,32 +37,43 @@ export default function useCart() {
   const shouldSkipAccountCartByAccountIdQuery = Boolean(!accountId || cartStore.hasAnonymousCartCredentials || isLoadingViewer || !shop || !shop._id);
   const shouldSkipAnonymousCartByCartIdQuery = Boolean(accountId || isLoadingViewer || !cartStore.anonymousCartId || !cartStore.anonymousCartToken);
 
-  const { loading: isLoading, data: cartData, fetchMore, refetch: refetchCart } = useQuery(accountCartByAccountIdQuery, {
-    skip: shouldSkipAccountCartByAccountIdQuery,
+  const [
+    fetchAccountCart,
+    { loading: isLoading, called: accountCartQueryCalled, data: cartData, fetchMore, refetch: refetchAccountCart }
+  ] = useLazyQuery(accountCartByAccountIdQuery, {
     variables: {
       accountId,
       shopId: shop && shop._id
     },
     pollInterval: shouldSkipAccountCartByAccountIdQuery ? 0 : 10000
   });
 
-  const { data: cartDataAnonymous, refetch: refetchCartAnonymous } = useQuery(anonymousCartByCartIdQuery, {
-    skip: shouldSkipAnonymousCartByCartIdQuery,
+
+  const [
+    fetchAnonymousCart,
+    { data: cartDataAnonymous, called: anonymousCartQueryCalled, refetch: refetchCartAnonymous }
```

Comment:
I think It was a good idea to rename `refetchCart` to `refetchAccountCart` to avoid confusion, but I would rename `refetchCartAnonymous` to `refetchAnonymousCart` accordingly.
```suggestion
    { data: cartDataAnonymous, called: anonymousCartQueryCalled, refetch: refetchAnonymousCart }
```